tests/kernel/mem_protect/demand_paging/testcase.yaml

```diff
@@ -6,7 +6,9 @@ tests:
       - kernel
       - mmu
       - demand_paging
-    platform_allow: qemu_x86_tiny
+    platform_allow:
+      - qemu_cortex_a53
```

Comment:
Maybe also add `qemu_corter_a53/smp` as this PR also enables SMP for demand paging.